app/models/state_change.rb

```diff
@@ -11,47 +13,55 @@ class StateChange < ApplicationRecord
   include Uuid::Uuidable
   include Asset::Ownership::ChangesOwner
 
-  belongs_to :user
-  validates :user, presence: true
+  attr_accessor :customer_accepts_responsibility
 
```

Comment:
belongs_to associations are required by default in Rails 5, but this functionality isn't enabled in SS. Slowly updating our belongs to as I come across them.